<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MAA2-5CG05165DH" failures="2" tests="2" name="Test" time="16.361" errors="0" timestamp="2024-10-10T00:27:53 IST">
  <testcase classname="testCasePackage.TC_003_Login_data_driven_test" name="verify_loginDDT" time="11.353">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCasePackage.TC_003_Login_data_driven_test.verify_loginDDT(TC_003_Login_data_driven_test.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCasePackage.TC_003_Login_data_driven_test" name="verify_loginDDT" time="12.84">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;org.openqa.selenium.TakesScreenshot.getScreenshotAs(org.openqa.selenium.OutputType)&amp;quot; because &amp;quot;takesScreenshot&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.TakesScreenshot.getScreenshotAs(org.openqa.selenium.OutputType)" because "takesScreenshot" is null
at testCasePackage.BaseClass.captureScreen(BaseClass.java:68)
at utilities.Extend_report.onTestFailure(Extend_report.java:81)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
</testsuite> <!-- Test -->
